src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
 // export const Person = ({ person }) => ();
+export const Person = ({ props }) => {
+  const sex = props.sex === 'f' ? 'husband' : 'wife';
```

Comment:
There is a logical error here. The condition should be `props.sex === 'm' ? 'wife' : 'husband';` to correctly assign the relationship title based on the sex.